components/logging.js

```diff
@@ -129,9 +132,14 @@ async function fetchLogsFromArchive(
     params: {
       "time_window.start_time.seconds": startTimeSeconds,
       "time_window.end_time.seconds": endTimeSeconds,
+      page_size: 50,
```

Comment:
This means Fleet Archive will return 50 results per call. The time taken for FA to return is related to the size of the result, so if 50 results is 1MB, and 500 results is 8MB, then it will take ~8x more time to return 500 results.

Reply:
This needs to be tested more, but it probably is more performant to have a large page size, because the query latency has some overhead.

Reply:
a larger page size generally gave better overall performance, but 50 is the max we can set for collect vehicle calls. one of my suggestions was to find a way for the Fleet Archive server to increase this limit.